libs/community/langchain_community/document_compressors/flashrank_rerank.py

```diff
@@ -69,11 +71,10 @@ def compress_documents(
         final_results = []
 
         for r in rerank_response:
-            metadata = r["meta"]
-            metadata["relevance_score"] = r["score"]
-            doc = Document(
-                page_content=r["text"],
-                metadata=metadata,
-            )
-            final_results.append(doc)
+            if r["score"] > self.score_threshold:
+                doc = Document(
+                    page_content=r["text"],
+                    metadata={"flash_rerank": {"id": r["id"], "relevance_score": r["score"]}, **r["metadata"]},
```

Comment:
This library is named `FlashrankRerank` please use this naming also.

```suggestion
                    metadata={"flashrank_rerank": {"id": r["id"], "relevance_score": r["score"]}, **r["metadata"]},
```